app/cafe.py

```diff
@@ -0,0 +1,19 @@
+import datetime
+from app.errors import (NotVaccinatedError,
+                        OutdatedVaccineError,
+                        NotWearingMaskError)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str:
+        if "vaccine" not in visitor:
+            raise NotVaccinatedError("The Person has not been vaccinated.")
+        elif visitor["vaccine"]["expiration_date"] < datetime.date.today():
```

Comment:
This is a code style issue. According to the checklist, you should use descriptive error messages. Here, the error message is not descriptive enough. You should specify why the vaccine is considered outdated. For example: 'The vaccine of the person has expired.'.